### PR TITLE
First implementation of scraping car data

### PR DESCRIPTION
**Pull request by Mchristos**
Basic implementation of spider scraping just the make and model from automart.co.za

### CODE REVIEW

File: car_spider.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    def parse(self, response):
+        for car in response.xpath("//div[@class='search-results']"):
+            make, model = car.xpath('div/div/p/em/text()').getall()
+            yield {
+                'make': make,
+                'model': model 
+            }
+
+        next_page = response.css('li.next a::attr("href")').get()
```
Comment (Mchristos) [inline car_spider.py]: This paging code doesn't work as expected

**Review by JemLukeBingham — CHANGES_REQUESTED** [review]
Last three lines of code appear not to be doing anything useful. Please either remove or get it working before merging.

**Comment by Mchristos** [comment]
Paging now functions. I'm not sure if it crashes on the final page though, as then there would be no "Next" button - should handle it but need to check.

**Comment by Mchristos** [comment]
Lmk what you think. I know checking `len()==2` is prob not ideal but not sure how else to do it (e.g. when I searched for "Fiat" cars on the site and used that URL instead, this data was length 1 and just had the model, no make!)

**Pull request merged**